MonoGame.Framework/MonoGame.Framework.Native.csproj

```diff
@@ -1,4 +1,4 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
I wonder why git is showing a diff here

Reply:
A whitespace change of some sort.  It happens.